torch/csrc/jit/codegen/cuda/root_domain_map.cpp

```diff
@@ -574,7 +574,27 @@ void ComputeAtRootDomainMapBuilder::mapPointwiseOrReductionOp(Expr* e) {
         TensorDomain::noReductions(i->getMaybeRFactorDomain());
     TORCH_INTERNAL_ASSERT(in_root.size() == out_root.size());
     for (size_t it = 0; it < in_root.size(); it++) {
-      setMaybeMapped(in_td, in_root[it], out_td, out_root[it]);
+      if (e->isA<WelfordOp>()) {
```

Comment:
Can you make this a doubly-nested loop, one for `e->inputs()` and another for `e->outputs()`? That way, it would work for any multi-output expressions.

Reply:
Updated. Thanks.